modules/administration-guide/partials/con_che-postgresql.adoc

```diff
@@ -1,21 +1,28 @@
 // Module included in the following assemblies:
 //
-// {prod-id-short}-workspace-controller
+// {prod-id-short}-workspace-controller-with-che-server
 
 [id="{prod-id-short}-postgresql_{context}"]
 = {prod-short} and PostgreSQL
 
-The PostgreSQL database is a prerequisite to configure {prod-short}. The {prod-short} administrator can choose to connect {prod-short} to an existing PostgreSQL instance or let the {prod-short} deployment start a new dedicated PostgreSQL instance.
+The PostgreSQL database is a prerequisite for {prod-short} server and {identity-provider}.
 
-The {prod-short} server uses the database to persist user configurations (workspaces metadata, Git credentials). {identity-provider} uses the database as its back end to persist user information.
+The {prod-short} administrator can choose to connect {prod-short} to: 
 
-[cols=2*]
-|===
-ifeval::["{project-context}" == "che"]
-| Source code
-| link:{link-postgres-dockerfile-location}[{prod-short} Postgres]
-endif::[]
+* An existing PostgreSQL instance. 
+* Let the {prod-short} deployment start a new dedicated PostgreSQL instance.
```

Comment:
If we connect the bullet points to the preceding sentence, it reads awkward to say, "The Che administrator can choose to connect Che to let the Che deployment start a new dedicated PostgreSQL instance." 

Consider a rephrase, like:
```
The Che administrator can choose to:

- Connect Che to an existing PostgreSQL instance. 
- Let the Che deployment start a new dedicated PostgreSQL instance.
```